frontend/app/components/CartridgeScoreboard.tsx

```diff
@@ -51,21 +123,25 @@ async function CartridgeScoreboard({cartridge_id}:{cartridge_id:string}) {
 
                                 <td scope="row" className="px-6 py-4 break-all">
```

Comment:
Since the address is being truncated, adding the "title" property will create a hover effect that displays the complete address. It will also be useful if we have nicknames or things like that.
```suggestion
                                <td scope="row" title={scoreInfo.user_address} className="px-6 py-4 break-all">
```